test/middleware.test.js

```diff
@@ -229,6 +229,17 @@ describe('middleware', () => {
           .expect(206, done);
       });
 
+      it('should return the "200" code for the "GET" request with malformed range header which is ignored', (done) => {
+        request(app).get('/bundle.js').set('Range', 'abc').expect(200, done);
+      });
```

Comment:
@evilebottnawi I think a 200 response code is fine since we are intentionally ignoring these header cases.

Reply:
Agree